LYNX-940 - Documentation update for SCP-B2B-1.0.2 release — DO NOT MERGE YET

cod40403 opened the pull request:
## Purpose of this pull request

This pull request (PR) adds documentation for the SCP B2B Package and updates the changelog for the SCP B2B 1.0.1, 1.0.2 release.

### Associated JIRA ticket
https://jira.corp.adobe.com/browse/LYNX-940
<!--OPTIONAL Add link to JIRA ticket associated with update.-->

danidelcar committed:
first pass for eds assets

bdenham committed:
WIP: Update navigation for B2B

bdenham committed:
WIP navigation improvements

bdenham committed:
fixed build

bdenham committed:
navigation reorganization

bdenham committed:
Merge branch 'develop' into reorganize-site-for-B2B

bdenham committed:
update developer docs organization

bdenham committed:
Merge branch 'reorganize-site-for-B2B' of https://github.com/commerce-docs/microsite-commerce-storefront into reorganize-site-for-B2B

cod40403 committed:
Code updated

deepak-soni1 committed:
Document updated

cod40403 committed:
LYNX-940 Flter added for new changelog section

cod40403 committed:
Code updated

bdenham committed:
udpated navigation/structure of Storefront Authoring docs and added overview topics

bdenham committed:
Restructure Quick Start sections with Overview pages

- Add Overview pages for Quick Start and Storefront Authors sections
- Rename Create your storefront to create-content for Storefront Authors
- Update navigation structure in astro.config.mjs
- Add appropriate redirects for backward compatibility
- Update sidebar styling for consistent color scheme
- Create Commerce Blocks overview page

bdenham committed:
Fix broken link in dropins/all/installing.mdx

- Update boilerplate template link from /get-started/#boilerplate-template to /get-started/boilerplate-project/
- Resolves link validation error after Quick Start section restructuring

bdenham committed:
Create missing create-storefront.mdx file

- Restore original 'Create a storefront' tutorial content from git history
- Fixes broken link in Quick Start navigation pointing to /get-started/create-storefront/
- Retrieved content from commit b78148c4 to ensure complete tutorial is preserved

bdenham committed:
Add redirects for restructured overview pages

- Add /get-started/overview/ → /get-started/ redirect
- Add /merchants/storefront-builder/create-your-content/ → /merchants/storefront-builder/create-content/ redirect
- Ensures backward compatibility for bookmarked links after content restructuring

bdenham committed:
removed file from templates directory

bdenham commented:
@cod40403 Please change the base branch of this PR to [b2b-nov-release](https://github.com/commerce-docs/microsite-commerce-storefront/tree/b2b-nov-release). Thanks!

bdenham committed:
Merge branch 'b2b-nov-release' into sidebar-and-overview-updates

cod40403 commented:
> @cod40403 Please change the base branch of this PR to [b2b-nov-release](https://github.com/commerce-docs/microsite-commerce-storefront/tree/b2b-nov-release). Thanks!

Hi @bdenham,
Should I update the target branch to [b2b-nov-release](https://github.com/commerce-docs/microsite-commerce-storefront/tree/b2b-nov-release)? I noticed that changing only the base branch will not include my changes in the PR.

cod40403 committed:
Merge branch 'develop' into LYNX-940

emipallares committed:
Add AddressValidation container to configuration and documentation

emipallares committed:
Enhance user account tutorials with new layout customization and address validation features. Added 'Customize the layout of the AddressForm' and 'Validate address in My Account' tutorials, updated configuration for tutorials section, and included a new address validation image.

bdenham committed:
remove topics out of configuration

bdenham committed:
more updates to topic organization and discoverability

bdenham committed:
fixed SDK links-for now

bdenham committed:
more draft templates and guidance

cod40403 committed:
Merge branch 'commerce-docs:develop' into LYNX-940

cod40403 committed:
Updated SCP-B2B 1.0.2 release notes

cod40403 committed:
Updated SCP-B2B 1.0.2 release notes

cctina516 committed:
add naming rules and new image

bdenham committed:
Add CLI tool for drop-in documentation generation

- Add dropin-docs-generator.js CLI script
- Add comprehensive README.md documentation
- Update package.json with new dropins command
- Update pnpm-lock.yaml with dependencies

bdenham committed:
Update drop-in documentation templates

- Refactor all template files to use placeholders instead of specific examples
- Add container-overview.mdx template with CardGrid navigation
- Create separate container templates (ContainerOne, ContainerTwo)
- Update all templates to use Steps component for numbered lists
- Add comprehensive placeholder system for developer customization
- Update astro.config.mjs for new template structure
- Add B2B overview page for drop-ins

bdenham committed:
added Diagram component to container templates

bdenham committed:
remove leftover registry file

bdenham committed:
changed remove to delete

bdenham committed:
changed CLI command name

bdenham committed:
update readme and alphabetize dropin section introduction

bdenham committed:
deleted old template files

bdenham committed:
Apply suggestion from @fnhipster

Co-authored-by: Carlos A. Cabrera <316104+fnhipster@users.noreply.github.com>

bdenham committed:
Apply suggestion from @fnhipster

Co-authored-by: Carlos A. Cabrera <316104+fnhipster@users.noreply.github.com>

bdenham committed:
Update _dropin-templates/dropin-installation.mdx

bdenham committed:
Merge pull request #462 from commerce-docs/templates-only

Update drop-in documentation templates

bdenham committed:
Merge branch 'cli-and-templates' into cli-only

bdenham committed:
Merge pull request #461 from commerce-docs/cli-only

Add CLI tool for drop-in documentation generation

bdenham committed:
fixed malformed comment from template reviews

bdenham committed:
Merge branch 'develop' into cli-and-templates

bdenham committed:
Merge branch 'develop' into cli-and-templates

devops-devdocs committed:
chore: publish sdk release elsie/1.5.0-alpha3002

bdenham committed:
Merge pull request #466 from commerce-docs/elsie/1.5.0-alpha3002

chore: publish SDK release elsie/1.5.0-alpha3002

bdenham committed:
Merge branch 'develop' into cli-and-templates

bdenham committed:
Merge branch 'develop' into sidebar-and-overview-updates

bdenham committed:
Merge branch 'develop' into update-site-creator

bdenham committed:
update component imports to fix build

bdenham committed:
added and updated redirects

bdenham committed:
Merge pull request #464 from commerce-docs/cli-and-templates

New CLI and templates integration

bdenham committed:
Fix sidebar section naming and B2B configuration

- Changed 'Drop-ins - B2C' to 'Drop-ins for B2C'
- Changed 'Drop-ins - B2B' to 'Drop-ins for B2B'
- Fixed B2B link from '/b2b/' to '/dropins-b2b/overview/'
- Fixed B2B label from 'B2B Overview' to 'Overview'
- Added '/b2b/**' to exclude list to prevent topic validation errors
- Build now passes successfully

bdenham committed:
Move Drop-ins SDK section after B2B section

- Repositioned Drop-ins SDK to come after Drop-ins for B2B
- This follows the logical flow: Overview → B2C → B2B → SDK
- Developers learn to use prebuilt drop-ins before building custom ones
- Build passes successfully

bdenham committed:
Move Multistore section to be right after Quick Start in Storefront Authors sidebar

- Repositioned Multistore section to come immediately after Quick Start
- New order: Quick Start → Multistore → Commerce Blocks → Content customizations
- This provides better logical flow for storefront authors
- Build passes successfully

bdenham committed:
Revert Multistore section to original position below Commerce Blocks

- Moved Multistore section back to its original position after Content customizations
- Restored original order: Quick Start → Commerce Blocks → Content customizations → Multistore
- Build passes successfully

bdenham committed:
wip navigation order and link fixes

bdenham committed:
Fix 404 error: Remove broken PDP Slots sidebar link

- Removed 'PDP Slots' link from Product Details sidebar section
- The corresponding slots.mdx file does not exist in the product-details directory
- Build now completes without 404 errors for this route

bdenham committed:
Fix 404 error: Update B2B Overview link to correct path

- Changed B2B Overview link from '/dropins-b2b/overview/' to '/b2b/'
- The actual B2B content is located at /b2b/ not /dropins-b2b/overview/
- Build now generates /b2b/index.html correctly without 404 errors

bdenham committed:
links checked and sidebar topics to sentence case

bdenham committed:
Update redirects based on recent file changes

- Remove broken redirect for /product-details/pdp-slots (file deleted)
- Add redirect for /get-started/create-storefront -> /get-started/create-storefront/
- Verify all existing redirects point to valid files
- Redirects now up to date with recent file changes from last 3 weeks

bdenham committed:
Fix broken internal link in multistore documentation

- Fix link from /setup/configuration/commerce-configuration/ to /setup/configuration/commerce-configuration
- The file exists as .mdx not as index.mdx in a directory
- This resolves a broken internal link found during link validation

bdenham committed:
Merge branch 'develop' into sidebar-and-overview-updates

bdenham committed:
fixed build errors

bdenham committed:
Merge branch 'sidebar-and-overview-updates' of https://github.com/commerce-docs/microsite-commerce-storefront into sidebar-and-overview-updates

bdenham committed:
fixed last of the build errors

bdenham committed:
warning fixes

bdenham committed:
removed visble B2B entry in sidebar

bdenham committed:
excluded b2b folder to fix build

bdenham committed:
revert the folder to remove the unnecessary underscore

bdenham committed:
Fixed insert feature for CLI after merge

bdenham committed:
fixed the CLI after merge

bdenham committed:
checked all links manually and updated

bdenham committed:
added sidebar links to fix build

bdenham committed:
Merge pull request #445 from commerce-docs/sidebar-and-overview-updates

Sidebar and overview updates

dependabot[bot] committed:
Bump the npm_and_yarn group across 1 directory with 4 updates

Bumps the npm_and_yarn group with 4 updates in the / directory: [astro](https://github.com/withastro/astro/tree/HEAD/packages/astro), [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite), [@babel/helpers](https://github.com/babel/babel/tree/HEAD/packages/babel-helpers) and [@babel/runtime](https://github.com/babel/babel/tree/HEAD/packages/babel-runtime).


Updates `astro` from 5.13.4 to 5.13.7
- [Release notes](https://github.com/withastro/astro/releases)
- [Changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
- [Commits](https://github.com/withastro/astro/commits/astro@5.13.7/packages/astro)

Updates `vite` from 7.1.3 to 7.1.5
- [Release notes](https://github.com/vitejs/vite/releases)
- [Changelog](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite/commits/v7.1.5/packages/vite)

Updates `@babel/helpers` from 7.28.3 to 7.28.4
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.28.4/packages/babel-helpers)

Updates `@babel/runtime` from 7.28.3 to 7.28.4
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.28.4/packages/babel-runtime)

---
updated-dependencies:
- dependency-name: astro
  dependency-version: 5.13.7
  dependency-type: direct:production
  dependency-group: npm_and_yarn
- dependency-name: vite
  dependency-version: 7.1.5
  dependency-type: direct:development
  dependency-group: npm_and_yarn
- dependency-name: "@babel/helpers"
  dependency-version: 7.28.4
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: "@babel/runtime"
  dependency-version: 7.28.4
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>

bdenham committed:
tweaks to organization of multistep and LB

bdenham committed:
remove B2C title until late

bdenham committed:
Merge branch 'develop' into sidebar-and-overview-updates

bdenham committed:
Merge pull request #471 from commerce-docs/sidebar-and-overview-updates

Sidebar and overview updates

bdenham committed:
duplicate documentation for create storefront

bdenham committed:
Merge branch 'develop' into dependabot/npm_and_yarn/npm_and_yarn-d236122b09

bdenham committed:
Merge pull request #470 from commerce-docs/dependabot/npm_and_yarn/npm_and_yarn-d236122b09

Bump the npm_and_yarn group across 1 directory with 4 updates

bdenham committed:
Merge branch 'develop' into duplicate-docs

bdenham committed:
Merge branch 'develop' into update-site-creator

bdenham committed:
Merge branch 'develop' into USF-2773-Account-Integrating-the-new-AddressValidation-container

bdenham committed:
Merge branch 'releases/b2b-nov-release' into LYNX-940

bdenham commented:
@cod40403 Yes, I changed the base branch to [b2b-nov-release](https://github.com/commerce-docs/microsite-commerce-storefront/tree/b2b-nov-release), assuming that the PR is for the B2B release. We on the docs team will merge the b2b integration branch into `develop` to conincide with the B2B release.

bdenham committed:
updated different docs page

bdenham committed:
updated image to latest

bdenham committed:
Merge pull request #457 from commerce-docs/update-site-creator

add naming rules and new image

bdenham committed:
Merge branch 'develop' into duplicate-docs

bdenham committed:
Merge pull request #473 from commerce-docs/duplicate-docs

duplicate documentation for create storefront

bdenham committed:
Merge branch 'develop' into USF-2773-Account-Integrating-the-new-AddressValidation-container

bdenham committed:
fix build

bdenham committed:
add tutorial sidebar links to fix build

bdenham committed:
Apply suggestion from @bdenham

bdenham committed:
Apply suggestion from @bdenham

bdenham committed:
Apply suggestion from @bdenham

bdenham committed:
Apply suggestion from @bdenham

bdenham committed:
Merge pull request #453 from commerce-docs/USF-2773-Account-Integrating-the-new-AddressValidation-container

USF-2773 Integrating the new AddressValidation container in the user-account drop-in

bdenham committed:
Update dropin packages

bdenham committed:
removed outdated dropin

bdenham committed:
Merge pull request #474 from commerce-docs/dropin-package-updates

Dropin package updates

bdenham committed:
add draft container docs for giftcard options

bdenham committed:
redirect old link to pdp containers

yutingjiangYTJ committed:
add description for compact design in Incrementer

yutingjiangYTJ committed:
Merge branch 'develop' into sdk-input-file

bdenham committed:
Merge pull request #476 from commerce-docs/pdp-redirect

redirect old link to pdp containers

bdenham committed:
Add container rendered image

bdenham committed:
resize the container image

bdenham committed:
Merge branch 'develop' into sdk-input-file

bdenham committed:
Update src/content/docs/sdk/components/incrementer.mdx

bdenham committed:
Merge pull request #477 from commerce-docs/sdk-input-file

add description for compact design in Incrementer

bdenham committed:
add working cartID

bdenham committed:
Merge pull request #478 from commerce-docs/fix-cart

add working cartID

bdenham committed:
AI generated extend-or-build docs

bdenham committed:
Merge branch 'gift-card-options' into extend-or-build

bdenham committed:
cut a bunch of content

bdenham committed:
Apply suggestions from code review

Co-authored-by: Stephen <sirugh@users.noreply.github.com>

bdenham committed:
remove invalid link

bdenham committed:
Update with more charts, rearrage

bdenham committed:
add critical info about enrichment.json file

bdenham committed:
removed unnecessary notes

bdenham committed:
Merge branch 'develop' into extend-or-build

bdenham committed:
Merge branch 'develop' into gift-card-options

bdenham committed:
added suggestion from review, refined, and removed redundences

bdenham committed:
removing giftcard options for preview

bdenham committed:
vastly simplified the topic

bdenham committed:
simplified and improved to focus on NOT

bdenham committed:
reorder sidebar nav promote extending and customizing droping over creating new

bdenham committed:
More focus on extening of creating

bdenham committed:
Merge pull request #475 from commerce-docs/gift-card-options

Adds container docs to PDP for giftcard options

bdenham committed:
Merge branch 'develop' into extend-or-build

bdenham committed:
pull vocab section back to top

bdenham committed:
Merge branch 'develop' into enrichment-additions

bdenham committed:
Merge pull request #480 from commerce-docs/enrichment-additions

Add critical info about enrichment.json file

bdenham committed:
Merge branch 'develop' into extend-or-build

danidelcar committed:
first iteration for aem assets eds

bdenham committed:
Nearly complete rewrite to incorporate new Slack discussion

danidelcar committed:
Added EDS info for AEM Assets integration

danidelcar committed:
Merge branch 'develop' into del-aem-assets

bdenham committed:
fixed formatting layoutof bullets, headings, text

bdenham committed:
Update src/content/docs/dropins/all/extend-or-create.mdx

Co-authored-by: Carlos A. Cabrera <316104+fnhipster@users.noreply.github.com>

bdenham committed:
updates from reviews

bdenham committed:
Merge branch 'extend-or-build' of https://github.com/commerce-docs/microsite-commerce-storefront into extend-or-build

bdenham committed:
Update src/content/docs/dropins/all/extend-or-create.mdx

Co-authored-by: Carlos A. Cabrera <316104+fnhipster@users.noreply.github.com>

bdenham committed:
updated from review feedback and added styling for scanning

bdenham committed:
fixed merge conflict

bdenham committed:
add link to discor, updated for consistency among sections

bdenham committed:
Reduced redundancies to shorten topic by about a third of the original

bdenham committed:
light editing for clarity and concision

meker12 committed:
Update instance ID references in commerce config

Add Environment-ID as required for ACO

bdenham committed:
Merge pull request #485 from commerce-docs/meker12-patch-2

Update instance ID references in commerce config

bdenham committed:
Merge branch 'develop' into extend-or-build

bdenham committed:
review with lots of updates

bdenham committed:
cutting and consolidating repetitive content

bdenham committed:
Merge branch 'develop' into del-aem-assets

bdenham committed:
Merge branch 'del-aem-assets' of https://github.com/commerce-docs/microsite-commerce-storefront into del-aem-assets

bdenham committed:
removed unused Callout component

bdenham committed:
changes to improve focus and clarity

bdenham committed:
updates to Tasks to align with Boilerplate integration

bdenham committed:
Fixed Callouts update

sirugh committed:
add links to config generator to configuration page

Signed-off-by: Stephen Rugh <rugh@adobe.com>

bdenham committed:
Update src/content/docs/setup/configuration/aem-assets-configuration.mdx

Co-authored-by: Stephen <sirugh@users.noreply.github.com>

bdenham committed:
Update src/content/docs/setup/configuration/aem-assets-configuration.mdx

Co-authored-by: Stephen <sirugh@users.noreply.github.com>

bdenham committed:
Updated based on review

bdenham committed:
updated location and wording of sections

bdenham committed:
minor wording fix

bdenham committed:
Update src/content/docs/setup/configuration/aem-assets-configuration.mdx

Co-authored-by: Stephen <sirugh@users.noreply.github.com>

bdenham committed:
Merge pull request #482 from commerce-docs/del-aem-assets

Adds section for AEM Assets integration in merchant docs

bdenham committed:
Merge branch 'develop' into extend-or-build

bdenham committed:
Merge pull request #479 from commerce-docs/extend-or-build

Extend-or-build docs

bdenham committed:
Update src/content/docs/setup/configuration/commerce-configuration.mdx

bdenham committed:
Merge branch 'develop' into cfg-gen-link

bdenham committed:
Merge pull request #486 from commerce-docs/cfg-gen-link

add links to config generator to configuration page

keoko committed:
address integration using last Google Maps API

keoko committed:
Merge branch 'develop' into USF-2761-address_autocomplete

bdenham committed:
Merge pull request #488 from keoko/USF-2761-address_autocomplete

USF-2761: address integration tutorial using last Google Maps API

sirugh committed:
major refactor of configuration page structure and content

Signed-off-by: Stephen Rugh <rugh@adobe.com>

bdenham committed:
Apply suggestions from code review

sirugh committed:
Update src/content/docs/setup/configuration/commerce-configuration.mdx

Co-authored-by: Margaret Eker <meker@adobe.com>

sirugh committed:
Update src/content/docs/setup/configuration/commerce-configuration.mdx

Co-authored-by: Margaret Eker <meker@adobe.com>

sirugh committed:
Update src/content/docs/setup/configuration/commerce-configuration.mdx

Co-authored-by: Margaret Eker <meker@adobe.com>

sirugh committed:
Update src/content/docs/setup/configuration/commerce-configuration.mdx

Co-authored-by: Margaret Eker <meker@adobe.com>

sirugh committed:
Remove customer group from config ref

sirugh committed:
apply feedback

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
add specific section for ACO analytics

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
remove all store header. Update policy to reference wildcard instead of attribute_code

Signed-off-by: Stephen Rugh <rugh@adobe.com>

sirugh committed:
update aco references

Signed-off-by: Stephen Rugh <rugh@adobe.com>

bdenham committed:
Update src/content/docs/setup/configuration/commerce-configuration.mdx

Co-authored-by: Margaret Eker <meker@adobe.com>

bdenham committed:
Merge pull request #491 from commerce-docs/root-cat-default

major refactor of configuration page structure and content

cod40403 committed:
Documentation update for SCP-4.8.8 and SCP-4.7.11 Releases

cod40403 committed:
Code updated

bdenham committed:
Add TableWrapper component and enhance Options/Option components

- Add TableWrapper component for easy markdown table styling with nowrap columns
- Enhance Options component to accept customizable label prop
- Update Option component to use dynamic label via CSS custom properties
- Improve Option badge width to auto-adjust based on label length

bdenham committed:
Merge pull request #498 from commerce-docs/component-enhancements

Add TableWrapper component and enhance Options/Option components

keharper committed:
Update changelog.mdx

keharper committed:
Update install.mdx

keharper committed:
Update v247.mdx

keharper committed:
Update v248.mdx

keharper committed:
Merge branch 'develop' into LYNX-962

keharper committed:
Merge pull request #496 from cod40403/LYNX-962

Documentation update for SCP-4.8.8 and SCP-4.7.11 Releases

keharper committed:
Revert docs for SCP 4.7.11

keharper committed:
Merge pull request #503 from commerce-docs/revert-496

Revert docs for SCP 4.7.11

cod40403 committed:
Conflicts fixed and code updated

cod40403 committed:
Merge branch 'develop' of github.com:cod40403/microsite-commerce-storefront into LYNX-940

cod40403 commented:
Closing this PR for a new PR with proper code.
